### PR TITLE
changes banana peel suicide to be more inspired

### PR DESCRIPTION
**Pull request by Bawhoppen**
the other one was too obvious

also I claimed this suicide 2 years ago

### CODE REVIEW

File: code/modules/hydroponics/grown/banana.dm [hunk 1]
```diff
@@ -48,10 +48,9 @@
 	throw_speed = 3
 	throw_range = 7
 
-/obj/item/weapon/grown/bananapeel/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is deliberately slipping on [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
-	playsound(loc, 'sound/misc/slip.ogg', 50, 1, -1)
```
Comment (Cyberboss) [inline code/modules/hydroponics/grown/banana.dm]: LEAVE THIS IN!!!

**Comment by Tacolizard** [comment]
why though

**Comment by GunHog** [comment]
I find this version inferior. The original one has humor and even a sound to go with it! This version seems to make even LESS sense than the first! At least if you try to slip yourself, you can realistically break your own neck in the process.

**Comment by xxalpha** [comment]
I don't think you got the joke, bawhoppen.

**Comment by Bawhoppen** [comment]
I got the joke
Look: I had this claimed #14228

**Comment by Tacolizard** [comment]
> the other one was too obvious

??what did he mean by this??

**Comment by Bawhoppen** [comment]
the joke was too obvious? this is shock humor

**Comment by Armhulen** [comment]
Spill out all the organs and then :+1:

**Comment by Bawhoppen** [comment]
I was going to give it a unique spooky sound but my headphones are broken so I can't test now
But I will close this for community good-will

**Pull request closed without merging**